packages/components/src/components/input/input.stories.ts

```diff
@@ -69,864 +35,269 @@ export const Default = {
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Size = {
+  render: () => html`
+    <div class="grid grid-cols-2 gap-12 content-end">
+      <sd-input size="lg" label="Label" placeholder="Large"></sd-input>
+      <sd-input size="md" label="Label" placeholder="Medium"></sd-input>
+      <sd-input size="sm" label="Label" placeholder="Small"></sd-input>
+    </div>
+  `
 };
 
 /**
- * Add descriptive help text to an input with the `help-text` attribute. For help texts that contain HTML, use the `help-text` slot instead.
+ * Use the `label` attribute to give the input an accessible label.
+ *
+ * For labels that contain HTML, use the `label` slot instead.
  */
 
-export const HelpText = {
-  name: 'Help Text',
-  args: overrideArgs([
-    { type: 'attribute', name: 'label', value: 'Nickname' },
-    { type: 'attribute', name: 'help-text', value: 'How would you like to be called?' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Label = {
+  render: () => html`
+    <div class="flex flex-row gap-12 items-baseline">
+      <sd-input label="Label Attribute" spellcheck></sd-input>
+      <sd-input spellcheck>
+        <div slot="label" class="text-lg">Label Slot</div>
+      </sd-input>
+    </div>
+  `
 };
 
 /**
- * Use the `placeholder` attribute to add a placeholder.
+ *Use the `placeholder` attribute to add a placeholder.
  */
-
-export const Placeholders = {
-  args: overrideArgs([{ type: 'attribute', name: 'placeholder', value: 'Type something' }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Placeholder = {
+  render: () => html`
+    <div class="w-[250px]">
+      <sd-input placeholder="Placeholder example" spellcheck></sd-input>
+    </div>
+  `
 };
 
 /**
- * Add the `clearable` attribute to add a clear button when the input has content.
+ * Use the `value` attribute to set the value.
  */
-
-export const Clearable = {
-  args: overrideArgs([{ type: 'attribute', name: 'clearable', value: true }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Value = {
+  render: () =>
+    html` <div class="w-[250px]">
+      <sd-input value="Value example"></sd-input>
+    </div>`
 };
 
 /**
- * Add the `password-toggle` attribute to add a toggle button that will show the password when activated. Only works with `type="password"`.
+ * Use the `inputmode` attribute Tells the browser what type of data will be entered by the user, allowing it to display the appropriate virtual keyboard on supportive devices.
```

Comment:
* Use the `inputmode` attribute tells the browser what type of data will be entered by the user, allowing it to display the appropriate virtual keyboard on supportive devices.

Reply:
The color of the disabled text should be lighter